tsconfig.json

```diff
@@ -12,7 +12,8 @@
     "outDir": "./dist/types",
     "target": "es6",
     "skipLibCheck": true,
-    "sourceMap": false
+    "sourceMap": false,
+    "lib": ["es6", "dom", "es2017"],
```

Comment:
What is the reason for "es2017"? Just curious about which features are used?

Reply:
Typescript underlines method .includes() as missing in es2016 when using it on array of strings. But you suggested above the better way to make calculation without .includes() . So, I'll remove that insertion back.